documentation/docs/pipelines/cloud-sdk/configuration.md

```diff
@@ -58,6 +58,10 @@ To configure static code checks, please configure the step `mavenExecuteStaticCo
 
 ### backendIntegrationTests
 
+The `backendIntegrationTests` stage has been integrated into the project "Piper" stage `Integration`.
+Thus, it is required to update the stage name in the stages section of your configuration to `integration`.
```

Comment:
Is the lowercase spelling of integration right?

Reply:
Yes it's the technical stage name.